backend/selenium_tests/people/test_people.py

```diff
@@ -0,0 +1,135 @@
+        assert "None" in pagePeopleDetails.getLabelResidenceStatus().text
+        assert "Western Sahara" in pagePeopleDetails.getLabelCountry().text
+        assert "-" in pagePeopleDetails.getLabelCountryOfOrigin().text
+        assert "-" in pagePeopleDetails.getLabelAddress().text
+        assert "-" in pagePeopleDetails.getLabelVilage().text
+        assert "-" in pagePeopleDetails.getLabelZipCode().text
+        assert "Hirat" in pagePeopleDetails.getLabelAdministrativeLevel1().text
+        assert "Adraskan" in pagePeopleDetails.getLabelAdministrativeLevel2().text
+        assert "-" in pagePeopleDetails.getLabelAdministrativeLevel3().text
+        assert "-" in pagePeopleDetails.getLabelAdministrativeLevel4().text
+        assert "-" in pagePeopleDetails.getLabelGeolocation().text
+        assert "Partial" in pagePeopleDetails.getLabelDataCollectingType().text
+        assert "Difficulty walking or climbing steps" in pagePeopleDetails.getLabelObservedDisabilities().text
+        assert "Some difficulty" in pagePeopleDetails.getLabelSeeingDisabilitySeverity().text
+        assert "A lot of difficulty" in pagePeopleDetails.getLabelHearingDisabilitySeverity().text
+        assert "None" in pagePeopleDetails.getLabelPhysicalDisabilitySeverity().text
+        assert "Cannot do at all" in pagePeopleDetails.getLabelRememberingOrConcentratingDisabilitySeverity().text
+        assert "Some difficulty" in pagePeopleDetails.getLabelpagePeopleDetailsCareDisabilitySeverity().text
+        assert "A lot of difficulty" in pagePeopleDetails.getLabelCommunicatingDisabilitySeverity().text
+        assert "Not Disabled" in pagePeopleDetails.getLabelDisability().text
+        assert "1427939569" in pagePeopleDetails.getLabelBirthCertificate().text
+        assert "Burundi" in pagePeopleDetails.getLabelIssued().text
+        assert "388855545" in pagePeopleDetails.getLabelDriverLicense().text
+        assert "Saint Barthélemy" in pagePeopleDetails.getLabelIssued().text
+        assert "1342079771" in pagePeopleDetails.getLabelElectoralCard().text
+        assert "French Southern Territories" in pagePeopleDetails.getLabelIssued().text
+        assert "1872704691" in pagePeopleDetails.getLabelNationalPassport().text
+        assert "Guinea" in pagePeopleDetails.getLabelIssued().text
+        assert "755043089" in pagePeopleDetails.getLabelNationalId().text
+        assert "Anguilla" in pagePeopleDetails.getLabelIssued().text
+        assert "771861294" in pagePeopleDetails.getLabelUnhcrId().text
+        assert "Saudi Arabia" in pagePeopleDetails.getLabelIssued().text
+        assert "898322765" in pagePeopleDetails.getLabelWfpId().text
+        assert "Liberia" in pagePeopleDetails.getLabelIssued().text
+        assert "-" in pagePeopleDetails.getLabelEmail().text
+        assert "Invalid Phone Number" in pagePeopleDetails.getLabelPhoneNumber().text
+        assert "-" in pagePeopleDetails.getLabelAlternativePhoneNumber().text
```

Comment:
```
def test_page_people_details(pagePeopleDetails):
    expected_values = {
        "getLabelAddress": "-",
        "getLabelVilage": "-",
        "getLabelZipCode": "-",
        "getLabelAdministrativeLevel1": "Hirat",
        "getLabelAdministrativeLevel2": "Adraskan",
        # ... add the rest of your labels here ...
        "getLabelIssued": "Saudi Arabia",
    }

    for label, expected_value in expected_values.items():
        assert expected_value in getattr(pagePeopleDetails, label)().text
```
        
        all this could be simplified